concepticondata/conceptlists/Lai-2023-291.tsv

```diff
@@ -0,0 +1,292 @@
+Lai-2023-291-134	97	the house	1252	HOUSE	494	the house
+Lai-2023-291-135	84	the head	1256	HEAD	74	the head
+Lai-2023-291-136	239	the wing	1257	WING	298	the wing
+Lai-2023-291-137	183	to sing	1261	SING	1184	to sing
+Lai-2023-291-138	90	high / tall	1265	HIGH	968	high / tall
+Lai-2023-291-139	138	narrow	1267	NARROW	977	narrow
+Lai-2023-291-140	168	the salt	1274	SALT	61	the salt
+Lai-2023-291-141	81	the hand	1277	HAND	107	the hand
+Lai-2023-291-142	227	to vomit	1278	VOMIT	1535	to vomit
+Lai-2023-291-143	379	to fall	1280	FALL		to fall
+Lai-2023-291-144	312	the fat (grease)	1282	FAT (FROM ANIMALS)		the fat (grease)
+Lai-2023-291-145	220	today	1283	TODAY	738	today
+Lai-2023-291-146	96	hot	1286	HOT	1062	hot
+Lai-2023-291-147	26	cold (of temperature)	1287	COLD	1063	cold (of temperature)
+Lai-2023-291-148	296	the back	1291	BACK		the back
+Lai-2023-291-149	5	bad	1292	BAD	1053	bad
+Lai-2023-291-150	202	the stick	1295	STICK	586	the stick
+Lai-2023-291-151	64	the foot	1301	FOOT	103	the foot
+Lai-2023-291-152	68	the fox	1312	FOX	325	the fox
+Lai-2023-291-153	128	the moon	1313	MOON	4	the moon
+Lai-2023-291-154	221	tomorrow	1329	TOMORROW	742	tomorrow
+Lai-2023-291-155	177	the sheep	1331	SHEEP	275	the sheep
+Lai-2023-291-156	140	the neck	1333	NECK	89	the neck
+Lai-2023-291-157	235	white	1335	WHITE	1006	white
+Lai-2023-291-158	46	to eat	1336	EAT	1198	to eat
+Lai-2023-291-159	151	the pig	1337	PIG	284	the pig
+Lai-2023-291-160	129	morning	1339	MORNING	749	morning // the morning
+Lai-2023-291-161	205	the sun	1343	SUN	2	the sun
+Lai-2023-291-162	328	the knife	1352	KNIFE		the knife // knife
+Lai-2023-291-163	111	to laugh	1355	LAUGH	1735	to laugh
+Lai-2023-291-164	106	the knee	1371	KNEE	101	the knee
+Lai-2023-291-165	223	the tooth (front)	1380	TOOTH	137	the tooth (front)
+Lai-2023-291-166	141	the needle (for sewing)	1382	NEEDLE (FOR SEWING)	578	the needle (for sewing)
+Lai-2023-291-167	121	the louse	1392	LOUSE	356	the louse
+Lai-2023-291-168	94	the horn (keratinized skin)	1393	HORN (ANATOMY)	263	the horn (keratinized skin)
+Lai-2023-291-169	18	the bone	1394	BONE	133	the bone
+Lai-2023-291-170	166	round	1395	ROUND	990	round
+Lai-2023-291-171	176	sharp	1396	SHARP	1020	sharp
+Lai-2023-291-172	40	dry	1398	DRY	1028	dry
+Lai-2023-291-173	39	to drink	1401	DRINK	1370	to drink
+Lai-2023-291-174	20	the breast (female)	1402	BREAST	94	the breast (female)
+Lai-2023-291-175	13	to bite	1403	BITE	1753	to bite
+Lai-2023-291-176	204	straight	1404	STRAIGHT	1003	straight
+Lai-2023-291-177	137	the name	1405	NAME	687	the name
+Lai-2023-291-178	50	far	1406	FAR	974	far
+Lai-2023-291-179	370	to breathe	1407	BREATHE		to breathe
+Lai-2023-291-180	85	to hear	1408	HEAR	1682	to hear
+Lai-2023-291-181	173	to see	1409	SEE	1471	to see
+Lai-2023-291-182	153	to play	1413	PLAY	1703	to play
+Lai-2023-291-183	414	to squeeze	1414	SQUEEZE		to squeeze
+Lai-2023-291-184	213	to think (reflect)	1415	THINK (REFLECT)	1726	to think (reflect)
+Lai-2023-291-185	409	to sit	1416	SIT		to sit
```

Comment:
Entry for `HUANG_1992_1820` column is empty. Just to confirm, is this not missing?

Reply:
These are new entries. This is correct.